ci/gpu/build.sh

```diff
@@ -40,6 +40,10 @@ conda list
 # BUILD - Build libcudf and cuDF from source
 ################################################################################
 
+# Temporarily install feather for testing
```

Comment:
Temporarily? Should this be eliminated? And there's no mention of feather except in this comment...

Reply:
This is for Python unit tests and this is in branch-0.6. This is "temporary" in that it's required for unit testing feather input / output on Pandas < 0.24. If we want to continue support for 0.23.4 then we should move this into the base image for CI.